README.md

```diff
@@ -1,6 +1,23 @@
 # COBalD/TARDIS Plugins for High Energy Physics
 
-This repository collects official and third-party plugins.
+[![Documentation Status](https://readthedocs.org/projects/cobald-hep-plugins/badge)](https://cobald-hep-plugins.readthedocs.io)
+
+This package collects official and third-party plugins.
```

Comment:
Please put here a reference to the cobald and Tardis repositories (and mention them explicitly), e.g.

```suggestion
This package collects official and third-party plugins for the [COBalD](/MatterMiners/cobald)/[TARDIS](/MatterMiners/tardis) software suite.
```